### DISCUSSION TITLE
ask before double import

### DISCUSSION THREAD

**Discussion by snickers2k** [mealie-recipes/mealie#3478] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

mealie does not notify about double importing

### Please provide a concise description of the feature that would resolve your issue.

would be great to be asked to be "sure?" before double importing the same recipe again

### Please consider and list out some caveats or tradeoffs made in your design decision

_

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by bbrodka2** [comment]
Yes, comparing existing recipe url with current import
Should be global option
Should be honored by single import, bulk import, and api import
Options:
1. Overwrite (Replace Current Recipe)
2. Skip (Keep current recipe))
3. Update Tags only
